Source/HelixToolkit.Wpf.SharpDX/Controls/DeferredRenderer.cs

```diff
@@ -1153,11 +1153,13 @@ public void Dispose()
         /// </summary>
         private class DeferredLightingVariables : IDisposable
         {
-            public DeferredLightingVariables(EffectsManager effectsContainer)
+            public DeferredLightingVariables(EffectsManager effectsContainer, IRenderTechniquesManager techniquesManager)

```

Comment:
For `DeferredLightingVariables` and `ScreenSpaceProcessingVariables`, we inject and `IRenderTechniquesManager` and an `IEffectsManager`.